src/plots/mapbox/layout_attributes.js

```diff
@@ -150,14 +148,68 @@ module.exports = {
             ].join(' ')
         },
 
+        circle: {
+            radius: {
+                valType: 'number',
+                dflt: 15,
+                role: 'style',
+                description: [
+                    'Sets the circle radius.',
+                    'Has an effect only when `type` is set to *circle*.'
+                ].join(' ')
+            },
+            color: {
+                valType: 'color',
+                dflt: defaultLine,
+                role: 'style',
+                description: [
+                    'Sets the circle color.',
+                    'Has an effect only when `type` is set to *circle*.'
+                ].join(' ')
+            }
```

Comment:
it's here: https://github.com/plotly/plotly.js/pull/681/files#diff-26aaae812b2edd7de7c8d4ab79df84f0R214